ngx_http_sxg_filter_module.c

```diff
@@ -604,6 +619,13 @@ static ngx_int_t ngx_http_sxg_header_filter(ngx_http_request_t* req) {
       }
     }
   }
+  // Add nosniff header entry.
+  if (!append_header(&req->headers_out.headers, "X-Content-Type-Options",
```

Comment:
Would it make more sense to put this inside `ngx_http_sxg_body_filter_impl`, where we add the `content-type` outer header? I'm trying to figure out how this will interact with the eventual fix to #59. e.g. If it involves modifying `ngx_http_sxg_body_filter_impl` to delete all the outer headers after adding them to the inner response, then it might delete this one (and also unnecessarily sign it).

But I don't have a good understanding of the difference between nginx header_filter and body_filter, and when it makes sense to put logic in one or the other. So I defer to your judgment entirely.

Reply:
Nice point. `ngx_http_sxg_body_filter_impl` would be better point. So I modified this to `ngx_http_sxg_body_filter_impl`.
So this PR fixes #59 too.